gateway-ha/pom.xml

```diff
@@ -19,6 +19,25 @@
         <app.main.class>io.trino.gateway.ha.HaGatewayLauncher</app.main.class>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.dropwizard</groupId>
+                <artifactId>dropwizard-dependencies</artifactId>
+                <version>${dropwizard.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>${dep.junit.version}</version>
```

Comment:
Is that necessary or does that come in with this version already ?

Reply:
I can get rid of this.